src/search/screens/search.screen.js

```diff
@@ -227,7 +230,13 @@ class Search extends Component {
         {searchStart &&
           !noUsersFound &&
           !noReposFound &&
-          <View style={styles.listContainer}>
+          <View
+            style={[
+              styles.listContainer,
+              (isPendingSearchUsers || isPendingSearchRepos) &&
```

Comment:
I like this. Maybe you can do `const isLoading = isPendingSearchUsers || isPendingSearchRepos` and later add that constant in here (it's a little more explicit)